### PR TITLE
LPS-137935

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-137935

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by miltonmc — CHANGES_REQUESTED** [review]
Just a hint to improve the fix:

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -633,6 +633,10 @@ div.ddm-tooltip-container {
 	}
 }
 
+div.ddm-first-row {
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:636]: Sweet! 🆒

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js [hunk 2]
```diff
@@ -158,7 +158,12 @@ export const Rows = ({children, rows}) => {
 	}
 
 	return rows.map((row, index) => (
-		<div key={index}>{children({index, row})}</div>
+		<div
+			className={classnames(index === 0 ? 'ddm-first-row' : '')}
+			key={index}
+		>
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js:164]: You got it

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:reevaluate:1287401_110

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by pyoo47** [comment]
ci:test:relevant

**Review by miltonmc — APPROVED** [review]
LGTM!

**Comment by miltonmc** [comment]
ci:forward

**Pull request closed without merging**